src/components/screens/wallet/explorer.js

```diff
@@ -19,6 +19,7 @@ const Wallet = ({
 }) => {
   const activeToken = useSelector(selectActiveToken);
   const { discreetMode } = useSelector(selectSettings);
+  const isDelegate = !!account.data.dpos?.delegate;
```

Comment:
Please use account.info[activeToken].summary.isDelegate. Delegate object will exist with empty information if isDelegate is false

Reply:
Done